src/TrueLayer/Mandates/MandatesApi.cs

```diff
@@ -66,6 +66,7 @@ public async Task<ApiResponse<CreateMandateResponse>> CreateMandate(CreateMandat
             );
         }
 
+        //TODO: is it correct that this method expects a mandate type?
```

Comment:
I don't think so, if you agree @tl-mauro-franchi let's leave this comment here

Reply:
Apparently, it is needed to get the [right auth token](https://docs.truelayer.com/docs/vrp-integration-checklist#33-generate-an-access_token)